README.md

```diff
@@ -14,13 +18,37 @@ connections and responses from end endpoint.
 
 (TODO: some words to go along with these diagrams)
 
-#### Development setup
+### Development setup
+
+#### [Node.js](http://nodejs.org)
+
+You may want to use [nvm](https://github.com/creationix/nvm) as your maanger of Node versions on your
```

Comment:
Before this, I think it might be useful to walk through instructions of checking to first see if they have node and npm installed/what versions they have, since that will negate some of these steps.